### DISCUSSION TITLE
Request: On the design page, move the svelte toolbar or make it movable

### DISCUSSION THREAD

**Discussion by cfepdan** [Budibase/budibase#5323] [Ideas]
When I create a container at the top of the page, the svelte toolbar covers up my container. I'd prefer that it were either smart enough to relocate, or that I could drag it somewhere more convenient. Here's a screenshot:

![image](https://user-images.githubusercontent.com/28074021/162535984-0a0220d5-09e3-43a1-a0e5-f6c260e36045.png)

**Comment by melohagan** [comment]
Hi @cfepdan 

I actually fixed this bug recently :)

I don't think it's been released yet though: https://github.com/Budibase/budibase/pull/5274

![Screenshot 2022-04-11 at 10 33 15](https://user-images.githubusercontent.com/101575380/162710307-9fb3a514-0e04-4a88-8af7-0d33195fb57f.png)
